runtime/vm/jfr.cpp

```diff
@@ -79,6 +79,11 @@ static int J9THREAD_PROC jfrSamplingThreadProc(void *entryArg);
 static void jfrExecutionSampleCallback(J9VMThread *currentThread, IDATA handlerKey, void *userData);
 static void jfrThreadCPULoadCallback(J9VMThread *currentThread, IDATA handlerKey, void *userData);
 static bool areJFRBuffersReadyForWrite(J9VMThread *currentThread);
+static void jfrThreadStatistics(J9VMThread *currentThread);
+static void jfrThreadContextSwitchRate(J9VMThread *currentThread);
+static void jfrClassLoadingStatistics(J9VMThread *currentThread);
+static void jfrCPULoad(J9VMThread *currentThread);
+static void jfrThreadCPULoad(J9VMThread *currentThread, J9VMThread *sampleThread);
 
 static UDATA jfrTypeIDHashFn(void *key, void *userData);
 static UDATA jfrTypeIDHashEqualFn(void *tableNode, void *queryNode, void *userData);
```

Comment:
Of all these forward declarations only these two are needed:
```c
static void jfrStartSamplingThread(J9JavaVM *vm);
static void initializeEventFields(J9VMThread *currentThread, J9JFREvent *jfrEvent, UDATA eventType);
```
Others are only referenced after their definition.